README.md

```diff
@@ -70,6 +70,97 @@ Congratulations- You are up and running!
 ### Cauldron via remote execution into AI Platform Notebook Instance
 Coming Soon - Our team at GMI uses Cauldron Notebooks in addition to Juptyer. 
 
+### PyCharm via remote execution into AI Platform Notebook Instance
+Prerequisites: 
+1. You must have Python 3.5+ installed
+2. You must have PyCharm installed with a PyCharm Professional Edition license 
+3. If using a windows machine, you must have PuTTY installed. 
+If using a Mac, you should have a built-in SSH client, so PuTTY is not needed.
+
+Helpful Links: 
+- Managing SSH keys in metadata-
+https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys
+- Creating a Remote Server Configuration in PyCharm-
+https://www.jetbrains.com/help/pycharm/creating-a-remote-server-configuration.html
+
+
+Instructions:
+1. On the AI Platform Notebook Instance you created,
+open your VM Instance details.
+On the dropdown for Remote access, select "SSH". 
+A linux terminal will pop-up in a new window.
+
+2. Follow the steps in the Managing SSH keys in metadata
+link above for creating a new SSH key.
+Example:
+- In the terminal, use the ssh-keygen command to generate a
+private SSH key file and a matching public SSH key file
+```
+ssh-keygen -t rsa -f ~/.ssh/[KEY_FILENAME] -C [USERNAME]
```

Comment:
In other parts of our documentation we use <> to indicate where a user needs to put custom inputs - Do you think [] makes more sense? Either way, it seems like it should be consistent within this repo.